exe/CMakeLists.txt

```diff
@@ -29,6 +37,8 @@ leatherman_install(puppet-db)
 
 # We duplicate the `help_test` as `smoke_test` until the default functionality
 # of the tool has been nailed down.
-add_test(NAME "smoke_test" COMMAND puppet-db)
-add_test(NAME "help_test" COMMAND puppet-db --help)
-add_test(NAME "version_test" COMMAND puppet-db --version)
+add_test(NAME "db_smoke_test" COMMAND puppet-db)
+add_test(NAME "db_help_test" COMMAND puppet-db --help)
+add_test(NAME "db_version_test" COMMAND puppet-db --version)
+add_test(NAME "query_help_test" COMMAND puppet-query --help)
+add_test(NAME "query_version_test" COMMAND puppet-query --version)
```

Comment:
Definitely, I think that might require a little more thought though as these tests also get run in appveyor (Windows)